spec/requests/api/requests_spec.rb

```diff
@@ -248,6 +248,31 @@
       expect(response.parsed_body).to match(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes allowed_tags in the request resources" do
```

Comment:
@abellotti @imtayadeway I have added tests in both places -- model and API.

If having both specs seem redundant, I can get rid of this commit d041b9624fc5b5da58eb95212371f3fd5479a217, which is the model spec.